### PR TITLE
Update help info

### LINKED ISSUE #63: update `python pvm.py --help`
at least it requires a clarification on the special meaning of port 8001, vs any other.

Will also need to be updated on the readme.

### PR DESCRIPTION
**Pull request by KaneBetter**
This fixes #63 .

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -120,7 +120,8 @@ usage: pvm.py [-h] [--port [PORT]]
 optional arguments:
   -h, --help     show this help message and exit
   --port [PORT]  The port that pvm.py will use to receive control messages.
-                 Default port is 8001
+                 Using the default port 8001 to control the first display.
+                 Using ny other port to control the second display.
```
Comment (omarcostahamido) [inline README.md:124]:
```suggestion
                 Use the default port 8001 to control the first display.
                 Use any other port to control the second display.
```

File: pvm.py [hunk 2]
```diff
@@ -196,7 +196,9 @@ def main(RECEIVE_PORT):
 	global PORT
 	global _logger
 	p = argparse.ArgumentParser()
-	p.add_argument('--port', type=int, nargs='?', default=8001, help='The port that pvm.py will use to receive control messages. Default port is 8001')
+	p.add_argument('--port', type=int, nargs='?', default=8001, help='The port that pvm.py will use to receive control messages. '
+																		'Using the default port 8001 to control the first display. '
+																		'Using ny other port to control the second display.')
```
Comment (omarcostahamido) [inline pvm.py]:
```suggestion
																		'Use the default port 8001 to control the first display. '
																		'Use any other port to control the second display.')
```

@KaneBetter you sure about these massive indents?

**Review by omarcostahamido — CHANGES_REQUESTED** [review]

**Review by omarcostahamido — APPROVED** [review]
@KaneBetter well, you know what to do.
I'll give you the green approve already, so you can merge it when you're done.
Thanks.

File: README.md (hunk 1 above)
Comment (KaneBetter) [reply README.md:124]: Done

File: pvm.py (hunk 2 above)
Comment (KaneBetter) [reply pvm.py]: Indentation fixed.

**Comment by omarcostahamido** [comment]
@KaneBetter you didn't fix the readme... could have simply accepted the proposed changes.

**Comment by KaneBetter** [comment]
I will fix the readme in #62 .

**Pull request merged**